packages/chakra-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -53,10 +54,11 @@ export default function BaseInputTemplate<
       isReadOnly={readonly}
       isInvalid={rawErrors && rawErrors.length > 0}
     >
-      {displayLabel && !!label && (
+      {labelValue(
```

Comment:
My making `labelValue()` take a component I saved a little bit of logic in many other places